apps/damap-frontend/src/app/components/layout/layout.component.ts

```diff
@@ -31,14 +46,63 @@ export class LayoutComponent implements OnInit {
     const browserLang = this.translate.getBrowserLang();
     this.translate.use(browserLang?.match(/en|de/) ? browserLang : 'en');
     this.lang = this.translate.currentLang.toUpperCase();
+
+    this.observer.observe([Breakpoints.Handset]).subscribe(result => {
+      this.isSmallScreen = result.matches;
+      this.checkScreenSize();
+    });
+  }
+
+  ngAfterViewInit(): void {
+    setTimeout(() => {
+      this.checkScreenSize();
+    }, 0);
+  }
+
+  @HostListener('window:resize')
+  onResize(): void {
+    this.debounce(() => this.checkScreenSize(), 300)();
+  }
+
+  private debounce(func: Function, wait: number): (...args: any[]) => void {
+    let timeout: any;
+    return (...args: any[]) => {
+      clearTimeout(timeout);
+      timeout = setTimeout(() => func.apply(this, args), wait);
+    };
+  }
+
+  private checkScreenSize(): void {
+    if (this.isSmallScreen) {
+      this.isCollapsed = true;
+      this.sidenav.mode = 'over';
```

Comment:
this will be set in the html template as well, depending on the `isSmallScreen` member